addons/flink-kubernetes-operator/parameter.cue

```diff
@@ -0,0 +1,14 @@
+parameter: {
+        //+usage=Deploy to specified clusters. Leave empty to deploy to all clusters.
+        clusters?: [...string]
+        //+usage=Namespace to deploy to, defaults to cert-manager
+        namespace: *"flink-operator" | string
+        // +usage=Specify if create  the webhook or not
+        "webhook.create": *false | bool
```

Comment:
```suggestion
        createWebhook: *false | bool
```
The rest are same. The reason for this is having dots in names will make it hard for users to input in CLI as Helm strvals.

Reply:
Good tips.